srcpkgs/micronucleus/template

```diff
@@ -0,0 +1,25 @@
+# Template file for 'micronucleus'
+pkgname=micronucleus
+version=2.04
+revision=1
+hostmakedepends="libusb-compat-devel"
+makedepends="libusb-devel libusb-compat-devel"
+short_desc="Upload tool for the Micronucleus avr bootloader"
+maintainer="Robert Lowry <bobertlo@gmail.com>"
+license="GPL-2.0-or-later"
+homepage="https://github.com/micronucleus/micronucleus"
+distfiles="https://github.com/micronucleus/micronucleus/archive/${version}.tar.gz"
+checksum=e09d4e427cde4dac43c3bac2e6f91903ae205263b1598beb8cef4f786692b506
+
+do_patch() {
```

Comment:
It's better to use `post_patch` here, else applying patches automatically from the `patches` dir doesn't work.

Reply:
Thanks!